Zend/zend_ast.c

```diff
@@ -1373,12 +1375,25 @@ static ZEND_COLD void zend_ast_export_attributes(smart_str *str, zend_ast *ast,
 			smart_str_appendc(str, ')');
 		}
 
-		if (newlines) {
-			smart_str_appendc(str, '\n');
-			zend_ast_export_indent(str, indent);
-		} else {
-			smart_str_appendc(str, ' ');
-		}
+		smart_str_appendc(str, ']');
```

Comment:
Also needs to be removed, closing `]` is written in `zend_ast_export_attributes()`.